### DISCUSSION TITLE
Doubt Regarding  Attempted At

### DISCUSSION THREAD

**Discussion by adhil-tinvio** [riverqueue/river#578] [Q&A]
Hi Team.
wanted to know if the attempted at also gets updated if a river_job fails and then retries

**Comment by bgentry** [comment]
Yes: https://github.com/riverqueue/river/blob/8669cbc38baf69eeda9dc4eb2944e8ef514d12c7/riverdriver/riverpgxv5/internal/dbsqlc/river_job.sql#L155

Going forward, please use the discussion section to ask questions so we can the issues section for bug tracking 🙏🏻
